subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -7113,3 +7416,421 @@ static uint8_t force_md_cnt_calc(struct lll_conn *lll_conn, uint32_t tx_rate)
 	return force_md_cnt;
 }
 #endif /* CONFIG_BT_CTLR_FORCE_MD_AUTO */
+
+
+/*
+ * NOTE: NEW EXPERIMENTAL LLCP CHANGES BELOW
```

Comment:
One could argue this comment is not needed, as LLCP refactored is by definition EXPERIMENTAL until further notice

Reply:
That's correct, i will remove it